src/components/DistanceMapView/index.android.js

```diff
@@ -31,6 +31,7 @@ function DistanceMapView(props) {
                 <View style={StyleUtils.combineStyles(styles.mapViewOverlay, props.overlayStyle)}>
                     <BlockingView
                         icon={Expensicons.EmptyStateRoutePending}
+                        iconColor={undefined}
```

Comment:
Already pointed out. Essentially, if we have the default prop value as undefined, then we can avoid repeatedly setting `iconColor` to undefined when we use the component.